python/helpers/pydev/_pydev_bundle/_pydev_calltip_util.py

```diff
@@ -0,0 +1,160 @@
+'''
+Id:          "$Id$"
+Copyright:   Copyright (c) 2015 Bank of America Merrill Lynch, All Rights Reserved
```

Comment:
I don't think this copyright is valid as the contribution is covered by CLA that transfers the copyright to JetBrains and assignes the Apache 2.0 license to the code.